arbos/programs/programs.go

```diff
@@ -283,6 +288,70 @@ func (p Programs) getProgram(codeHash common.Hash, time uint64) (Program, error)
 	return program, err
 }
 
+// SaveActiveProgramToWasmStore is used to save active stylus programs to wasm store during rebuilding
+func (p Programs) SaveActiveProgramToWasmStore(statedb *state.StateDB, codeHash common.Hash, code []byte, time uint64, debugMode bool, rebuildingStartBlockTime uint64) error {
```

Comment:
move this function to a separate file, make it only be compiled in native with:
```
//go:build !wasm
// +build !wasm
```
and remove the activateProgramInternal stub function from wasm.